install_scripts/templates/kubernetes/init.sh

```diff
@@ -216,6 +221,7 @@ initKube() {
     local kubeV=$(kubeadm version --output=short)
 
     # init is idempotent
+    CURRENT_KUBERNETES_VERSION=$(cat /etc/kubernetes/manifests/kube-apiserver.yaml | grep image: | grep -oE '[0-9]+.[0-9]+.[0-9]')
```

Comment:
its weird you check if this file exists right after this line

Reply:
I'll add a comment